crates/types/src/error.rs

```diff
@@ -28,6 +29,14 @@ pub enum Error {
     IoError(#[from] std::io::Error),
     #[error(transparent)]
     FromHexError(#[from] starknet_rs_core::types::eth_address::FromHexError),
+    #[error(transparent)]
+    TransactionExecutionError(#[from] TransactionExecutionError),
+    // TODO import cairo-lang-starknet to the project so this error could be created with its
```

Comment:
this todo will be addressed in another task?

Reply:
yes

Reply:
is it created already?

Reply:
no

Reply:
usually, what we do in that situation is creating new task during PR phase

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/issues/198